docs/docs-content/automation/crossplane/crossplane.md

```diff
@@ -0,0 +1,35 @@
+---
+sidebar_label: "Crossplane"
+title: "Crossplane"
+description: "Learn how to use Crossplane with Palette and Palette VerteX."
+hide_table_of_contents: false
+sidebar_position: 0
+tags: ["crossplane", "iac", "automation", "infrastructure as code"]
+---
+
+Palette supports the open-source Cloud Native Computing Foundation (CNCF) project
+[Crossplane](https://www.crossplane.io/). Crossplane transforms Kubernetes clusters into universal control planes,
+extending the Kubernetes API and enabling infrastructure resource provisioning and management across major
+infrastructure providers.
+
+These resources, called
+[Managed Resources (MR)](https://docs.crossplane.io/latest/concepts/managed-resources/#managed-resource-fields) within
+the Crossplane environment, are essentially Kubernetes Custom Resource Definitions (CRDs) that represent infrastructure
+resources as native Kubernetes objects. As Kubernetes objects, you can use standard commands like `kubectl describe` to
+interact with them. When users create a managed resource, Crossplane interacts with the infrastructure provider API to
```

Comment:
What is "them" in this context? The MR? Or the CRD? I understand they are both the same thing but to help the reader it would help to be explicit :)